### PR TITLE
Fixes `_create_custom_request_arguments`

### PR DESCRIPTION
**Pull request by pem70**
**What It Does**
Fixed a bug on `_create_custom_request_arguments` where changes on `custom_args` will stay after the function returns [#299]

**How to Test**

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [x] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

File: CHANGELOG.md
Comment (traeok) [inline CHANGELOG.md:10]: Looks like lines 9-10 can be removed 🙂

**Review by traeok — COMMENTED** [review]
LGTM, will approve once changelog is addressed. Thanks @pem70

**Review by traeok — APPROVED** [review]

File: CHANGELOG.md
Comment (zFernand0) [inline CHANGELOG.md:10]:
```suggestion
```

**Review by zFernand0 — APPROVED** [review]
The changes make sense to me 😋 

Hey @john-craig,
would you like to test this PR before it's merged? 😋

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @pem70!

**Pull request merged**
